sdk/src/main/java/com/oursky/authgear/latte/Latte.kt

```diff
@@ -88,4 +88,19 @@ class Latte(val authgear: Authgear, val customUIEndpoint: String) {
             authgear.fetchUserInfo()
         }
     }
+
+    suspend fun resetPassword(extraQuery: List<Pair<String, String>>): LatteHandle<Unit> {
+        val entryUrl = "$customUIEndpoint/recovery/reset"
+        val redirectUri = "latte://reset-complete"
```

Comment:
Do we need to update iOS SDK? It is now using HTTP scheme URL.

Reply:
I've changed it in ios sdk too
https://github.com/authgear/authgear-sdk-ios/blob/2023C01/Sources/Latte%2BFlows.swift#L144